src/components/media-views.js

```diff
@@ -943,12 +943,15 @@ AFRAME.registerComponent("media-video", {
           this.el.object3D.getWorldPosition(positionA);
           this.el.sceneEl.camera.getWorldPosition(positionB);
           const distance = positionA.distanceTo(positionB);
+          const globalRolloffFactor = window.APP.store.state.preferences.globalRolloffFactor;
```

Comment:
Instead of assuming that `globalRolloffFactor` is between 1 and 0, I'd just do a clamp on it

Reply:
Thanks, I did it.

Reply:
Preferences can be `undefined` if a user hasn't specified a pref. You should set a default in this case with something like:
```js
const pref = window.APP.store.state.preferences.globalRolloffFactor
const globalRolloffFactor = pref === undefined ? 1 : pref;
```
I see below that you are setting the preference when the audio system starts up, but it would be bad to rely on initialization order to prevent this bug. If we elevate this into the preference menu (instead of having it be a slash command) then we will also remove the code that resets it on startup. (https://github.com/mozilla/hubs/pull/2766/files#diff-02e07dc2b8c1d9775319e5e3806b1aaeR55) 

We do this for example here: https://github.com/mozilla/hubs/blob/24d7e9327483531b22883f88206459614207cd55/src/components/media-views.js#L474

Reply:
Sorry to un-resolve the above conversation with a different concern related to this same line. I was just worried you wouldn't have seen it otherwise.